decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -44,6 +44,23 @@ class Meeting < Meetings::ApplicationRecord
       scope :visible_meeting_for, lambda { |user|
         (all.distinct if user&.admin?) ||
           if user.present?
+            user_role_queries = %w(conference assembly participatory_process).map do |participatory_space_name|
```

Comment:
Hi! This is causing an error when the conferences module is not included. It should check if `Decidim::Conferences` is defined to include it in the array. Thanks!

Reply:
@ivan-mr @tramuntanal this affects these pr also:
https://github.com/decidim/decidim/pull/6925
https://github.com/decidim/decidim/pull/6933

Reply:
There's a `rescue` statement that's catching it no?

Reply:
The defined? method only works with Decidim::Conference object not with the "Decidim::Conference" string. First of all, I have to transform the string into a constant, but if I constantize the string and Decidim::Conference is not defined, it will raise an error.
Because of that, I put a begin rescue block to catch all modules that are optionals and not defined.
If you have found some execution errors from this part of code, can you explain how to reproduce it?
Thanks

Reply:
if we use a `rescue` we will ignore the error but assmelby and participaotry process will be ignored.
Couldn't we do something like this?

```ruby
if user.present?
            spaces = %w(assembly participatory_process)
            spaces.append("conference") if defined?(Decidim::Conferences)
            user_role_queries = spaces.map do |participatory_space_name|
```